src/index.ts

```diff
@@ -14,19 +14,35 @@ import { WebsocketService } from './services/websocket_service';
     const app = express();
     const server = app.listen(config.HTTP_PORT, () => {
         logger.info(
-            `Standard relayer API (HTTP) listening on port ${config.HTTP_PORT}!\nConfig: ${JSON.stringify(
+            `API (HTTP) listening on port ${config.HTTP_PORT}!\nConfig: ${JSON.stringify(
                 config,
                 null,
                 2,
             )}`,
         );
     });
-    const meshClient = new WSClient(config.MESH_WEBSOCKET_URI);
-    const orderWatcherService = new OrderWatcherService(meshClient);
-    await orderWatcherService.syncOrderbookAsync();
-    // tslint:disable-next-line:no-unused-expression
-    new WebsocketService(server, meshClient);
+    let meshClient: WSClient | undefined;
+    try {
+        meshClient = new WSClient(config.MESH_WEBSOCKET_URI);
+        const orderWatcherService = new OrderWatcherService(meshClient);
+        await orderWatcherService.syncOrderbookAsync();
+        // tslint:disable-next-line:no-unused-expression
+        new WebsocketService(server, meshClient);
+    } catch (err) {
+        logger.error(err);
+    }
     const orderBookService = new OrderBookService(meshClient);
     // tslint:disable-next-line:no-unused-expression
     new HttpService(app, orderBookService);
-})().catch(logger.error.bind(logger));
+})().catch(error => logger.error(error));
+
+process.on('uncaughtException', err => {
+    logger.error(err);
```

Comment:
does there need to be an `if(err)` check here as well?

Reply:
No the error is guaranteed to be there.